website/markdown/docs/commands/graph.mdx

```diff
@@ -45,8 +45,8 @@ tuist graph --format dot
       default: '`png`',
     },
     {
-      long: '`--layout-algorithm`',
-      short: '`-l`',
+      long: '`--algorithm`',
```

Comment:
do you think that `algorithm` is enough to understand that is the algorithm to change the layout of the graph?

Reply:
I haven't been thinking about it. I have just copied values from implementation. :/ I have wanted to fix documentation in this PR :)

Reply:
<img width="596" alt="Zrzut ekranu 2020-09-1 o 15 39 21" src="https://user-images.githubusercontent.com/4774319/91858320-503b3480-ec69-11ea-93ef-48a568c73bc3.png">

Reply:
oh ok :) got it, thanks for fixing the doc also on this one then!

Reply:
IMO, `--algorithm` is good enough if you ask me :)